app/models/reaction.rb

```diff
@@ -139,6 +139,7 @@ def contradictory_mod_reactions(category:, reactable_id:, reactable_type:, user:
   # - reaction is negative
   # - receiver is the same user as the one who reacted
   # - reaction status is marked invalid
+  # - reaction is not in a category that should be notified
   def skip_notification_for?(_receiver)
```

Comment:
At some point in the future, we ought to refactor this method: `_receiver` is required but isn't used, which also makes the method name and explanatory comment a bit misleading.